src/sortStudents.ts

```diff
@@ -1,16 +1,51 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+const getAverage = (value : number[]) : number => value
+  .reduce((prev, curr) => prev + curr, 0) / value.length;
```

Comment:
better use more meaningful variable names.
```suggestion
const getAverage = (grades : number[]) : number => grades
  .reduce((prev, curr) => prev + curr, 0) / value.length;
```
Instead of `prev, curr` better use `sum, grade` it would be more obvious for this function.